Signal/src/ViewControllers/ConversationView/ConversationViewModel.m

```diff
@@ -1276,6 +1333,28 @@ - (nullable NSNumber *)findGroupIndexOfThreadInteraction:(TSInteraction *)intera
     return @(groupIndex);
 }
 
+- (void)typingIndicatorStateDidChange:(NSNotification *)notification
+{
+    OWSAssertIsOnMainThread();
+
+    self.typingIndicatorsRecipient = [self.typingIndicators typingIndicatorsForThread:self.thread];
+}
+
+- (void)setTypingIndicatorsRecipient:(nullable NSString *)typingIndicatorsRecipient
```

Comment:
recipient->recipientId? 

No biggied, but it's kind of confusing terminology, because it's the *sender* of the typing message, not the recipient. The local user is actually the recipient, right?

Reply:
Done.  We've mostly standardized around "recipient id" for user/account ids in the codebase, so I was trying to be consistent with that.